src/docfx/build/redirection/RedirectionProvider.cs

```diff
@@ -16,14 +16,13 @@ internal class RedirectionProvider
         private readonly DocumentProvider _documentProvider;
         private readonly MonikerProvider _monikerProvider;
         private readonly BuildScope _buildScope;
+        private readonly BuildOptions _buildOptions;
         private readonly Func<PublishUrlMap> _publishUrlMap;
 
-        private readonly IReadOnlyDictionary<FilePath, string> _redirectUrls;
-        private readonly HashSet<PathString> _redirectPaths;
-        private readonly Lazy<(IReadOnlyDictionary<FilePath, FilePath> renameHistory,
-            IReadOnlyDictionary<FilePath, (FilePath, SourceInfo?)> redirectionHistory)> _history;
+        private readonly Watch<(Dictionary<FilePath, string> urls, HashSet<PathString> paths, RedirectionItem[] items)> _redirects;
+        private readonly Watch<(Dictionary<FilePath, FilePath> rename, Dictionary<FilePath, (FilePath, SourceInfo?)> redirect)> _history;
```

Comment:
```suggestion
        private readonly Watch<(Dictionary<FilePath, FilePath> renames, Dictionary<FilePath, (FilePath, SourceInfo?)> redirects)> _history;
```

to align with the above line